dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -37,18 +37,19 @@ public class StepwisePlanner
     public StepwisePlanner(
         IKernel kernel,
         StepwisePlannerConfig? config = null,
-        string? prompt = null)
+        string? prompt = null,
+        PromptTemplateConfig? promptUserConfig = null)
     {
         Verify.NotNull(kernel);
         this._kernel = kernel;
 
         this.Config = config ?? new();
         this.Config.ExcludedSkills.Add(RestrictedSkillName);
 
-        var promptConfig = new PromptTemplateConfig();
+        var promptConfig = promptUserConfig ?? new PromptTemplateConfig();
         var promptTemplate = prompt ?? EmbeddedResource.Read("Skills.StepwiseStep.skprompt.txt");
         string promptConfigString = EmbeddedResource.Read("Skills.StepwiseStep.config.json");
```

Comment:
This `EmbeddedResource` should only be read from file if the `promptUserConfig` arg was not passed in.

Reply:
Make sense. Updated